lib/private/Preview/Movie.php

```diff
@@ -144,6 +144,8 @@ private function generateThumbNail(int $maxX, int $maxY, string $absPath, int $s
 		$returnCode = -1;
 		$output = "";
 		if (is_resource($proc)) {
+			stream_set_blocking($pipes[1], 0);
+			stream_set_blocking($pipes[2], 0);
```

Comment:
```suggestion
			stream_set_blocking($pipes[1], false);
			stream_set_blocking($pipes[2], false);
```

Second parameter should be `bool`.